### ISSUE TITLE
Broken get_possible_topologies

### ISSUE THREAD

**Issue by Old-Shatterhand** [BojarLab/glycowork#57]
Based on an example, I rerun it on the latest state of the dev-branch and got an empty list.
```
glycan = "{6S}Neu5Ac(a2-6)GlcNAc(b1-3)Gal(b1-4)GlcNAc(b1-6)[Neu5Ac(a2-3)GlcNAc(b1-3)]GalNAc"
print("\n".join(graph_to_string(g) for g in get_possible_topologies(glycan)))
```
The expected output is
```
Neu5Ac6S(a2-6)GlcNAc(b1-3)Gal(b1-4)GlcNAc(b1-6)[Neu5Ac(a2-3)GlcNAc(b1-3)]GalNAc
Neu5Ac(a2-6)GlcNAc(b1-3)Gal(b1-4)GlcNAc6S(b1-6)[Neu5Ac(a2-3)GlcNAc(b1-3)]GalNAc
Neu5Ac(a2-6)GlcNAc(b1-3)Gal(b1-4)GlcNAc(b1-6)[Neu5Ac(a2-3)GlcNAc6S(b1-3)]GalNAc 
```
but I get
```
eu5Ac(a2-6)GlcNAc(b1-3)Gal6S(b1-4)GlcNAc(b1-6)Neu5Ac(a2-3)GlcNAc(b1-3))GalNAc
eu5Ac(a2-6)GlcNAc(b1-3)Gal(b1-4)GlcNAc6S(b1-6)Neu5Ac(a2-3)GlcNAc(b1-3))GalNAc
eu5Ac(a2-6)GlcNAc(b1-3)Gal(b1-4)GlcNAc(b1-6)Neu5Ac(a2-3)GlcNAc6S(b1-3))GalNAc
```
which is invalid (`eu5AC` instead of `Neu5Ac`), lacks the branching of the original glycan and puts the 6S at different positions (see the first in the list).

I'll try to find a fix, but might need help of a more experienced glycoworker.

**Comment by Old-Shatterhand** [comment]
After some debugging, I found the following: 

- The differences in the groups becoming sulfated lie in different `modification_map`s in `glycan_data/loader.py`.
- Both the invalid starting string and the missing brackets come from changes in `graph_to_string_int` since Commit [f60c6d4](https://github.com/BojarLab/glycowork/commit/f60c6d4d261a444a9b6ca29fc17b6323fdd4ee01) to the `dev` branch

**Comment by Old-Shatterhand** [comment]
This is going to be fixed by #59

**Comment by Bribak** [comment]
#59 has been merged and fixes this (will be in glycowork 1.4)
